runtime/list.go

```diff
@@ -170,6 +170,40 @@ func listCount(f *Frame, args Args, kwargs KWArgs) (*Object, *BaseException) {
 	return seqCount(f, args[0], args[1])
 }
 
+func listRemove(f *Frame, args Args, kwargs KWArgs) (*Object, *BaseException) {
+	if raised := checkMethodArgs(f, "remove", args, ListType, ObjectType); raised != nil {
+		return nil, raised
+	}
+	l := toListUnsafe(args[0])
+	l.mutex.Lock()
+	value := args[1]
+	found := false
+	var raised *BaseException
+	for i, elem := range l.elems {
+		var eq *Object
+		eq, raised = Eq(f, elem, value)
```

Comment:
Minor style suggestion: put these assignments directly in the if statement:

```
if eq, raised = Eq(f, elem, value); raised != nil {
```

Ditto for the next if statement.